wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-container fluid fill-height justify-center>
+    <v-card>
+      <v-card-title class="display-3 justify-center">
+        Leader Board
+      </v-card-title>
+      <v-card-text>
+        <v-simple-table>
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th style="text-align: center"># Games</th>
+              <th style="text-align: center">Avg. Attempts</th>
+              <th style="text-align: center">Avg. Seconds</th>
```

Comment:
```suggestion
              <th class="text-center"># Games</th>
              <th class="text-center">Avg. Attempts</th>
              <th class="text-center">Avg. Seconds</th>
```

Avoid using inline styles & instead take advantage of either pre-built CSS classes from vuetify, or your own CSS classes.